python/pyarrow/tests/test_fs.py

```diff
@@ -1326,7 +1326,10 @@ def test_py_open_append_stream():
         f.write(b"data")
 
 
+# TODO(kszucs): the following two cases fail inside non-default
+# AWS regions e.g. us-east-2
 @pytest.mark.s3
+@pytest.mark.skip
```

Comment:
Should this be reverted before merging? 
(I thought @xhochy encountered a similar some time ago?)

Reply:
Nice catch, definitely.